tests/test_visualization.py

```diff
@@ -150,6 +150,29 @@ def test_plot_targets(self):
         plot_targets(target_position, target_radius, (-2, 2, -2, 2, -2, 2), ax=ax)
         savefig(write_dir, filename)
 
+    def test_plot_circles(self):
+
+            # Draw four outer targets and one center target
+            filename = 'circles.png'
+            target_position = np.array([
+                [0, 0, 0],
+                [1, 1, 0],
+                [-1, 1, 0],
+                [1, -1, 0],
+                [-1, -1, 0]
+            ])
+            target_radius = 0.1
+            target_color = 'b'
+            plt.figure()
+            plot_circles(target_position, target_radius, (-2, 2, -2, 2))
```

Comment:
you've got (-2, 2, -2, 2) going into circle_color here, that's why you're getting an error i think

Reply:
good catch, thank you